### ISSUE TITLE
Cannot read property 'length' of undefined

### ISSUE THREAD

**Issue by toyorg** [mrmlnc/vscode-remark#7]
Using VSCode 1.10.2 I'm getting this error.

>[Remark]
>TypeError: Cannot read property 'length' of undefined

**Comment by mrmlnc** [comment]
Do you use plugins?

**Comment by toyorg** [comment]
Yes, but I disabled all of them and nothing changed.

**Comment by mrmlnc** [comment]
You can provide your `remark` settings? Most likely your issue related to https://github.com/mrmlnc/vscode-remark#about-error-handling.

**Comment by toyorg** [comment]
With this or without still the same.
```
remark.format":` {
        "rules": {
        "commonmark": true
    }}
```

**Comment by mrmlnc** [comment]
Thank you for the clarification. Looks like a bug. I'll work on it very soon.

Last question: If you specify this config:

```js
remark.format":` {
  "rules": {
    "commonmark": true
  },
  "plugins": []
}
```

The error continues to appear?

**Comment by toyorg** [comment]
Well, it's weird.

> 1. Open file only - remark don't work
> 2. Open folder with same file - remark is working

https://gfycat.com/HonestUntidyAgama

**Comment by toyorg** [comment]
I think there is a problem:
`return vscode.workspace.findFiles('**/*remarkrc', '**/node_modules/**').then((files) => {`

**Comment by toyorg** [comment]
There is fix for that bug #8

**Comment by mrmlnc** [comment]
Should be fixed in `1.1.0`.
